app/queries/internal/moderators_query.rb

```diff
@@ -14,7 +14,7 @@ def self.call(relation: User.all, options: {})
                      role_id_for(:trusted),
                    ).order("users.comments_count DESC")
                  else
-                   relation.joins(:users_roles).
+                   relation.joins(:roles).
```

Comment:
Is this change due to a deprecation?

Reply:
Nope, it just doesn't work without that change. Not totally sure why, though it works in current master as well if I change it to `:roles` 🤷

Reply:
`roles` is the actual association table and `users_roles` the join table, so I think it actually makes more sense like this anyway.